tests/integration/io/test_s3_reads_include_path.py

```diff
@@ -27,16 +27,15 @@ def test_read_multi_parquet_from_s3_with_include_file_path_column(minio_io_confi
     with minio_create_bucket(minio_io_config, bucket_name=bucket_name):
         file_paths = (
             daft.from_pydict(data)
-            .into_partitions(3)
-            .write_parquet(f"s3://{bucket_name}", io_config=minio_io_config)
+            .write_parquet(f"s3://{bucket_name}", partition_cols=["a"], io_config=minio_io_config)
```

Comment:
Changing `into_partitions` to instead have the write itself be partitioned. For swordfish.